offer-manager-domain/src/main/groovy/life/qbic/business/projects/create/CreateProjectOutput.groovy

```diff
@@ -0,0 +1,19 @@
+package life.qbic.business.projects.create
+
+import life.qbic.business.UseCaseFailure
+
+/**
+ * Output interface for the <span>Create Project</span> use case.
+ *
+ * @since 1.0.0
+ */
+interface CreateProjectOutput extends UseCaseFailure {
+
+    /**
+     * Called when a project has been successfully created.
+     *
+     * @param project The project that has been created in QBiC's data management system.
+     */
```

Comment:
```suggestion
    /**
     * Called when a project has been successfully created.
     *
     * @param project The project that has been created in QBiC's data management system.
     * @since 1.0.0
     */
```

Reply:
What kind of project is used here? I can't see an import statement for the `Project` object